src/test/java/rv/atqc/gttesting/possibility_view_main_page/PossibilityViewMainPageChrome.java

```diff
@@ -0,0 +1,38 @@
+package rv.atqc.gttesting.possibility_view_main_page;
+
+import java.util.concurrent.TimeUnit;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import io.github.bonigarcia.wdm.ChromeDriverManager;
+
+public class PossibilityViewMainPageChrome{
+
+	public WebDriver driver;
+	
+	@Test
+	public void checkExistenceOfTheTitle(){
+        driver.manage().timeouts().implicitlyWait(1, TimeUnit.SECONDS);
```

Comment:
I suggest using the explicit waiter... please find the links in the study_res channel